README.md

```diff
@@ -20,48 +21,82 @@ function sum(a, b) {
   return a + b;
 } `); // define a function
 
-console.log(await vm.call("sum", 2, 3)) // 5
+console.log(await vm.call("sum", 2, 3)); // 5
 
 vm.close(); // terminate the worker
 ```
 
-Permissions
------------
+## Default permissions
```

Comment:
`Default permissions` -> `CLI permissions`. It's unclear what does "default" mean here.